lms/djangoapps/verify_student/views.py

```diff
@@ -1210,6 +1224,10 @@ def post(self, request, course_id, checkpoint_name, location):
             try:
                 redirect_url = get_redirect_url(course_key, usage_key)
             except (ItemNotFoundError, NoPathToItem):
+                log.warning(
```

Comment:
i'm assuming that this is at warning because, though it's basically an error if this happens, it's neither totally uncommon nor catastrophic for the end user (since there's a sane default landing place).